packages/apidom-ls/src/config/asyncapi/target-specs.ts

```diff
@@ -0,0 +1,18 @@
+export const AsyncAPI200 = [{ namespace: 'asyncapi', version: '2.0.0' }];
+export const AsyncAPI210 = [{ namespace: 'asyncapi', version: '2.1.0' }];
+export const AsyncAPI220 = [{ namespace: 'asyncapi', version: '2.2.0' }];
+export const AsyncAPI230 = [{ namespace: 'asyncapi', version: '2.3.0' }];
+export const AsyncAPI240 = [{ namespace: 'asyncapi', version: '2.4.0' }];
+export const AsyncAPI250 = [{ namespace: 'asyncapi', version: '2.5.0' }];
+export const AsyncAPI260 = [{ namespace: 'asyncapi', version: '2.6.0' }];
+
+export const AsyncAPI2 = [
+  ...AsyncAPI200,
+  ...AsyncAPI210,
+  ...AsyncAPI220,
+  ...AsyncAPI230,
+  ...AsyncAPI240,
+  ...AsyncAPI250,
+  ...AsyncAPI260,
+];
+export const AsyncAPI = [...AsyncAPI2];
```

Comment:
In light of https://github.com/swagger-api/apidom/commit/1e15838314e7b2eea7ffa4d8259e25f41b9a2d64 can we remove this?

The reasoning is that we'd rather add target spec explicitly to what applies, then automatically activating existing rules for example for AsyncAPI 3.0 (when we're adding support for it). This total group of all AsyncAPI spec targets has no practical usage and causes more issues than it solves.